crates/holochain/src/core/workflow/genesis_workflow.rs

```diff
@@ -265,86 +247,3 @@ mod tests {
         }
     }
 }
-
-/* TODO: update and rewrite as proper rust docs
```

Comment:
Interestingly there were traces of DPKI in the codebase that dated back 5 years.